packages/create-bison-app/template/src/components/auth/LoginForm.tsx

```diff
@@ -91,11 +90,11 @@ export function LoginForm() {
         </PasswordField>
       </div>
       <div className="flex justify-between">
-        <NextLink href="/reset-password">
-          <Button variant="link" size="sm">
```

Comment:
I don't think `NextLink` is necessary anymore. I'm also not seeing it in the "pages" docs